### PR TITLE
title,description and icon

### PR DESCRIPTION
**Pull request by Nicat60**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="We are Azerbaijan based innovative design & development studio that offers web based products. We aim to deliver the best results, therefore, we approach every project with special care and we pay attention to every single detail so that we deliver what we promised"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo.png" />
```
Comment (azadsarxanli) [inline public/index.html]: bunu yeni logo ile deyisdir
ve `image/x-icon` olan versiyani da elave et bu sadece apple OS olanlar ucundur

File: public/index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <meta name="theme-color" content="#000000" />
```
Comment (azadsarxanli) [inline public/index.html]: theme coloru ise `prefers-color-scheme` hansidirsa, onunla deyisdir. light mode istifade eden telefonda/komputerde de dark (black) gorunmesi qeribe goruner

**Review by azadsarxanli — CHANGES_REQUESTED** [review]

**Review by zeyno12 — COMMENTED** [review]
everything is looking great

File: public/index.html [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
+    <link rel="icon" href="%PUBLIC_URL%/newLogo.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="We are Azerbaijan based innovative design & development studio that offers web based products. We aim to deliver the best results, therefore, we approach every project with special care and we pay attention to every single detail so that we deliver what we promised"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo.png" />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/newLogo.png" />
```
Comment (azadsarxanli) [inline public/index.html:13]: that's nice that you added `newLogo.png`, but you still forgot to add 
favicon for non-apple users:
please read the below article (example part): 
https://www.w3schools.com/html/html_favicon.asp

**Review by azadsarxanli — CHANGES_REQUESTED** [review]
- prefers color scheme meta tag ucun elave olunmalidi
- favicon non-apple istifadeciler ucun elave olunmalidi (https://www.w3schools.com/html/html_favicon.asp)

**Review by azadsarxanli — APPROVED** [review]

**Pull request merged**